tests/framework/unit_tests/TSA/testFunctionTransformers.py

```diff
@@ -0,0 +1,264 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  This Module performs Unit Tests for the TSA.ZeroFilter class.
+  It can not be considered part of the active code but of the regression test system
+"""
+import os
+import sys
+import numpy as np
+
+# add RAVEN to path
+ravenDir =  os.path.abspath(os.path.join(*([os.path.dirname(__file__)] + [os.pardir]*4)))
+frameworkDir = os.path.join(ravenDir, 'framework')
+if ravenDir not in sys.path:
+  sys.path.append(ravenDir)
+
+from ravenframework.utils import xmlUtils
+
+from ravenframework.TSA import LogTransformer, ArcsinhTransformer, SigmoidTransformer, TanhTransformer
+
+print('Modules undergoing testing:')
+testedClasses = [LogTransformer, ArcsinhTransformer, SigmoidTransformer, TanhTransformer]
+print(*testedClasses)
+print('')
+
+results = {"pass":0, "fail":0}
+
+def updateRes(comment, dtype, res, value, expected, printComment=True):
+  """
+    Updates results dictionary
+    @ In, comment, string, a comment printed out if it fails
+    @ In, dtype, type, the type of the value
+    @ In, res, bool, the result of the check
+    @ In, value, object, the value to compare
+    @ In, expected, object, the expected value
+    @ In, printComment, bool, optional, if False then don't print comment
+  """
+  if res:
+    results['pass'] += 1
+  else:
+    results['fail'] += 1
+    if printComment:
+      print(f'checking {str(dtype)} {comment} | {value} != {expected}')
+
+def checkFloat(comment, value, expected, tol=1e-10, update=True):
```

Comment:
FWIW, I've really wanted to move to using pytest and standardize all these checks across RAVEN unit tests, but have never had the bandwidth to do it. It's not required (by any means) for this work, but if there's spare bandwidth and interest, it would improve the unit tests dramatically.